doc/GettingStarted.md

```diff
@@ -0,0 +1,259 @@
+# Getting Started with ACON (Attested Containers)
+
+## Building ACON Components
+**Note**: Examples in this section assumes all ACON related files to be installed/stored in `~/acon`, which could be replaced by any directory path deemed suitable by the user.
+
+### Building and Testing the Guest Kernel
+
+You will need a working *aTD* (ACON TD) before launching ACON *Image*s. Booting an *aTD* requires a guest kernel and an initrd image.
+
+1. See [Building Linux Kernel](TDGuest.md#building-linux-kernel) for instructions on configuring and building the guest kernel.
```

Comment:
Currently, clicking on this link gets a '404 - Page Not Found' error. I am unsure whether this issue will persist after merging the pull request.

Reply:
I double-checked the link and it worked on my end. I'm not sure if it's because it is referring a new link that exists only in the PR (not in `main` yet). Guess the problem would go away after merge.